java-to-zod-gradle-plugin/src/test/resources/build.gradle.kts

```diff
@@ -0,0 +1,62 @@
+import cz.habarta.typescript.generator.*
+
+plugins {
+    id("java")
+    id("cz.habarta.typescript-generator") version "3.2.1263"
+    id("java-to-zod-gradle-plugin") version "0.7.0-SNAPSHOT"
+    id("com.github.node-gradle.node") version "3.0.1"
+}
+
+node {
+    version = "16.13.0"  // Or the version you want to use
+    download = true  // This ensures a local copy of Node.js is used
+}
+
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+dependencies {
+    implementation("cz.habarta.typescript-generator:typescript-generator-core:3.2.1263")
+    implementation("jakarta.validation:jakarta.validation-api:3.1.0")
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
+}
+
+tasks.withType<sh.ivan.zod.GenerateZodSchemas> {
+    val tempOutputDir = layout.buildDirectory.dir("temp/generated-schemas")
+    outputFile = tempOutputDir.map { it.file("schemas.ts") }.get().asFile
+    pluginParameters.jsonLibrary = JsonLibrary.jackson2
+    pluginParameters.classes = mutableListOf("sh.ivan.zod.resources.TestPersonClass", "sh.ivan.zod.resources.TestPersonRecord")
+}
```

Comment:
Is this the typical way to configure a plugin like this in Gradle? I'm expecting something more like this: https://github.com/vojtechhabarta/typescript-generator/blob/main/sample-gradle/build.gradle#L44-L65